renderlib/Fuse.cpp

```diff
@@ -11,8 +12,7 @@
 void
 Fuse::fuse(const ImageXYZC* img,
            const std::vector<glm::vec3>& colorsPerChannel,
-           const GradientData* channelGradientData,
-           const float* channelIsLabels,
+           const VolumeDisplay& volumeDisplay,
```

Comment:
changes in this file are just because the colormaps, label info, and gradient data are all stored in VolumeDisplay